core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -262,13 +273,16 @@ def predict_proba(self, X, normalize=None, **kwargs):
     def _predict_proba(self, X, normalize=False, **kwargs):
         return self.predict_proba(X=X, normalize=normalize, **kwargs)
 
-    def score_with_oof(self, y):
+    def score_with_oof(self, y, weights=None):
```

Comment:
as a general question, not sure how this is handled in other packages, but should we name it `weights, weights_val` or `w, w_val`, considering we have `y, y_val`.

Reply:
I'm in favor of w, w_val internally, since it is shorter while still clear.

Reply:
Naming is now: `sample_weights, sample_weights_val` for the model fit kwargs, and left as it was for internal trainer/learner functions to be updated as we see fit when documenting those functions.